export.go

```diff
@@ -67,7 +68,7 @@ func exportPostsZip(u *User, posts *[]PublicPost) []byte {
 			filename += p.Slug.String + "_"
 		}
 		filename += p.ID + ".txt"
-		files = append(files, exportedTxt{filename, p.Content, p.Created})
+		files = append(files, exportedTxt{filename, p.Title.String, p.Content, p.Created})
```

Comment:
just `p.Title`

Reply:
`p.Title` is a `zero.String` (not a `string`), so this get the correct value.

Reply:
ok, first time I've seen the package

Reply:
yep, it's just like `sql.NullString` but handles things differently for JSON parsing.